commands/skills/appraise.py

```diff
@@ -0,0 +1,142 @@
+from commands.command import MuxCommand
```

Comment:
To keep imports as consistent as possible, could this instead be 

`from evennia import default_cmds`

and then in the command below, 

`class CmdAppraise(default_cmds.MuxCommand):`